packages/test/end-to-end-tests/src/test/consensusOrderedCollectionEndToEndTests.spec.ts

```diff
@@ -4,7 +4,7 @@
  */
 
```

Comment:
can you add a simple test in this package that checks the container dirty state. doesn't need to be exhaustive, just enough to know it works on the happy path